cheroot/errors.py

```diff
@@ -64,15 +72,27 @@ def plat_specific_errors(*errnames):
     socket_errors_to_ignore.extend(plat_specific_errors('EPROTOTYPE'))
     socket_errors_nonblocking.extend(plat_specific_errors('EPROTOTYPE'))
 
+sock_shutdown_errs = (OSError, _OpenSSL_SysCallError)
+write_shutdown_errs = (OSError, _OpenSSL_SysCallError, ConnectionError)
 
 acceptable_sock_shutdown_error_codes = {
+    errno.EBADF,
```

Comment:
We can just inline this so it's not detached in some separate place in the module.
```suggestion
    errno.EBADF,
    *((errno.WSAENOTSOCK,) if IS_WINDOWS else ()),
```